src/main/java/com/hanzereversiai/projectp3/AppLauncher.java

```diff
@@ -1,5 +1,7 @@
 package com.hanzereversiai.projectp3;
 
+import com.hanzereversiai.projectp3.networking.Network;
```

Comment:
This import can be left out.

Reply:
We might as well run a cleanup every once in a while rather than bothering to check import statements every merge